webproject/src/main/java/com/google/sps/services/implementations/GetJoinServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps.services.implementations;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import com.google.appengine.api.users.User;
+import com.google.sps.proto.GetJoinProto.GetJoinRequest;
+import com.google.sps.services.interfaces.GetJoinService;
+import com.google.sps.authentication.AuthenticationHandlerSupplier;
+
+public class GetJoinServiceImpl implements GetJoinService {
+    @Override
+    public String execute(GetJoinRequest getJoinRequest) throws IOException {
+        String roomId = getJoinRequest.getRoomId();
+        String userEmail = AuthenticationHandlerSupplier.getAuthenticationHandler().getCurrentUser().getEmail();
+
+        StringBuilder url = new StringBuilder("https://summer20-sps-47.firebaseio.com/UserRoom.json?orderBy=%22userEmailRoom%22&equalTo=%22");
```

Comment:
this should be modified to use firebase API as example here:
https://firebase.google.com/docs/database/admin/retrieve-data

Reply:
done